src/Injector.ts

```diff
@@ -16,8 +16,8 @@ class Injector {
    * Render the base template and inject content if provided
    */
   injectAndRender (inputPath: string, inputIndexYaml: string, boatsRc: BoatsRC): string {
-    const fullPath = path.join(process.cwd(), inputPath);
-    const relativePathToRoot = path.relative(path.dirname(inputPath), path.dirname(inputIndexYaml));
+    const fullPath = upath.join(upath.toUnix(process.cwd()), inputPath);
```

Comment:
Not 100% sure it's required to wrap the `process.cwd` fall inside the toUnix call in all cases, but it's working at the moment like this.

Also, there's 4 occurances of this pattern `upath.toUnix(process.cwd())` - so probably should pull it out into a helper function.